webapp/src/components/Meeting.tsx

```diff
@@ -34,6 +39,9 @@ const Meeting = ({ meetingId }: MeetingProps) => {
   const currentParticipantId = meeting.participants.find(
     ({ principal_id }) => principal_id === principalId
   )?.id;
+  const [greatestSortOrderNumber] = meeting.meeting_notes
```

Comment:
The meeting notes are already sorted when they come from the API. Also, it's not supposed to be based on all the notes, only the ones from the agenda of the current user.

Reply:
To clarify, this only needs to be a linear operation at worst. Sorting is not linear.

Reply:
Do you meant that the sort order should be based on the meeting notes with `authoring_partcipant_id`'s that match the current principal's ID?

Reply:
Participant ids and principal ids are not comparable. Issue #171 specifies:

> the `sort_order` is set to be 1 plus the `sort_order` of the last item in the current user's agenda (the list of notes with the `participant_id` associated with the `principal_id` of the current user).

Reply:
Thanks!  The code was changed.